client/src/app/routes/nova/ValidatorsPage.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import TruncatedId from "~/app/components/stardust/TruncatedId";
+import { useValidators } from "~/helpers/nova/hooks/useValidators";
+import "./ValidatorsPage.scss";
+
+const ValidatorsPage: React.FC = () => {
+    const { validators, error } = useValidators();
+
+    // Build rank by stake map
+    const addressToRankByStake = new Map<string, number>();
+    const validatorsSortedByPoolStake = [...(validators ?? [])].sort((a, b) => (a.validator.poolStake > b.validator.poolStake ? -1 : 1));
```

Comment:
shouldnt the sorting come from the backend api? sorting in the frontend can lead to problems because we need to exhaust the whole cursor in order to sort properly

Reply:
i would just use the default sorting the backend gives us which i understand is based on poolStake and then we can implement more custom sortings later, wdyt?